src/solver/toolkit.c

```diff
@@ -412,6 +412,75 @@ EXPORT_TOOLKIT int  swmm_getSimulationParam(SM_SimSetting type, double *value)
     return error_code;
 }
 
+EXPORT_TOOLKIT int DLLEXPORT swmm_hotstart(SM_HotStart type, const char *hsfile)
```

Comment:
remove `DLLEXPORT` from the function declaration for this to work with the SWIG wrapper.